src/Sticky.jsx

```diff
@@ -170,17 +170,23 @@ class Sticky extends Component {
     }
 
     handleResize (e, ae) {
+        if (this.props.shouldFreeze()) { return; }
+
         winHeight = ae.resize.height;
         this.updateInitialDimension();
         this.update();
     }
 
     handleScrollStart (e, ae) {
+        if (this.props.shouldFreeze()) { return; }
+        
         scrollTop = ae.scroll.top;
         this.updateInitialDimension();
     }
 
     handleScroll (e, ae) {
+        if (this.props.shouldFreeze()) { return; }
```

Comment:
probably, 
but not sure if that will cause scrolling performance on start if we setState for virtual DOM comparison?